src/test/java/org/fundacionjala/sfdc/stepdefinition/LogInSteps.java

```diff
@@ -32,7 +32,7 @@ public void openTheFirefoxAndLaunchTheApplicationEnterTheUsernameAndPasswordAndP
     @Then("^I login to the application$")
     public void iLoginToTheApplication() {
         System.out.println("Click login.");
-        Assert.assertTrue(login.getCloudIcon());
+//        Assert.assertTrue(login.getCloudIcon());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)